sympy/concrete/summations.py

```diff
@@ -988,6 +988,24 @@ def eval_sum_symbolic(f, limits):
 
         r = gosper_sum(f, (i, a, b))
 
+        # to find Sum at points where denominator approaches to zero
+        from sympy import together, fraction, Mul, Add, ordered
+        if isinstance(r, (Mul,Add)) and not(any(x in r.atoms(Symbol) for x in limits)):
```

Comment:
Does this mean that `r` does not have any symbols in common with the limit? Do you mean `r.free_symbols`? Are all parts of a limit symbols?

Reply:
Oh, actually I was trying to avoid making condition on symbols in `limit`. Changed it and did `var = [x for x in var if x not in limits]` instead.